.github/workflows/archive-design-issues-in-projects.yml

```
@@ -0,0 +1,27 @@
+name: Archive Issue in Project
+
+on:
+  issues:
+    types: [labeled]
+
+jobs:
+  process-labeled-issue:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "16"
```

Is there a reason this needs to be Node v16? If not can you change it to:

```suggestion
          node-version-file: package.json
```

---

I think I did this because the action showing a warning that was saying something about moving to a later version of node. I just bumped it to be whatever it was recommending at the time. I'll change it to the suggestion.